document/src/tests/documentcalculatortestcase.cpp

```diff
@@ -170,22 +114,17 @@ DocumentCalculatorTest::testFieldNotFound() {
     Document doc(*_testRepo.getDocumentType("testdoctype1"), DocumentId("doc:test:foo"));
     doc.setValue(doc.getField("hfloatval"), FloatFieldValue(3.0));
     doc.setValue(doc.getField("headerlongval"), LongFieldValue(2));
-    try {
-        calc.evaluate(doc, std::move(variables));
-        CPPUNIT_ASSERT(false);
-    } catch (const vespalib::IllegalArgumentException&) {
-        // OK
-    }
+    EXPECT_THROW(calc.evaluate(doc, std::move(variables)),
+            vespalib::IllegalArgumentException);
```

Comment:
Mega-nitpick: 4 instances of mis-aligned indentation for `EXPECT_THROW`

Reply:
Fixed :)